src/olympia/translations/tests/test_models.py

```diff
@@ -701,6 +702,57 @@ def test_external_text_link(self, get_outgoing_url_mock):
         assert doc('b')[0].text == 'markup'
 
 
+class PurifiedMarkdownTranslationTest(TestCase):
```

Comment:
nit: Typically I think adding `Test` to the beginning is better than the end as this class now matches the class it is testing in a find/replace

<img width="416" alt="image" src="https://github.com/user-attachments/assets/5164e1f3-f111-4500-9e92-accccf00efc7" />